applications/mne_scan/plugins/ftbuffer/ftbuffer.cpp

```diff
@@ -146,6 +146,9 @@ bool FtBuffer::stop()
 
     m_bIsConfigured = false;
 
+    requestInterruption();
+    wait(500);
```

Comment:
Can you try if this also works without the 500ms wait?

Reply:
It seems to work fine without the wait command. Escaping the main run loop is pretty fast, so maybe it doesn't affect it with what we are doing now. The [wait call](https://doc.qt.io/qt-5/qthread.html#wait) does stop blocking early if the function returns quickly (which it seems to).